trigger/conf/global_settings.py

```diff
@@ -294,6 +294,9 @@
 # Assign NETDEVICES_SOURCE to NETDEVICES_FILE for backwards compatibility
 NETDEVICES_FILE = NETDEVICES_SOURCE
 
+# TextFSM Template Path. Commando will attempt to match a given show command with a template within this folder.
+TEXTFSM_TEMPLATE_DIR = os.getenv('TEXTFSM_TEMPLATE_DIR', os.path.join(PREFIX, 'vendor/ntc_templates'))
```

Comment:
Can you please add an entry in `docs/usage/configuration.rst` for this under the `Twister Settings` section explaining how it's used, and perhaps also link to the ansible-ntc templates we're currently using for this?